MagnetScannerIOS/ScannerGeo.swift

```diff
@@ -0,0 +1,76 @@
+//
+//  LocationServices.swift
+//  Magnet
+//
+//  Created by Francisco Jordano on 17/10/2016.
+//
+
+import Foundation
+import CoreLocation
+import SwiftyJSON
+
+class ScannerGeo: NSObject, CLLocationManagerDelegate, Scanner {
+  let lManager: CLLocationManager = CLLocationManager()
+  var callback: ((Dictionary<String, AnyObject>) -> Void)!
+  var initialized: Bool = false;
+  
+  init(callback: (Dictionary<String, AnyObject>) -> Void) {
+    super.init()
+    self.callback = callback
+  }
+  
+  func start() {
+    startLocationManager()
+  }
+  
+  func stop() {
+    lManager.stopUpdatingLocation()
```

Comment:
Where's the part where we keep telling it to scan in the background even after `.stop()` has been called?